src/rc_allocator.c

```diff
@@ -769,50 +769,61 @@ rc_allocator_assert_noleaks(rc_allocator *al)
 void
 rc_allocator_print_stats(rc_allocator *al)
 {
-   int64 divider = GiB / al->cfg->io_cfg->extent_size;
-   platform_default_log(
-      "----------------------------------------------------------------\n");
-   platform_default_log(
-      "| Allocator Stats                                              |\n");
-   platform_default_log(
-      "|--------------------------------------------------------------|\n");
-   uint64 curr_gib = al->stats.curr_allocated / divider;
+   // clang-format off
+   const char *dashes = "-------------------------------------------------------------------";
+   platform_default_log("|%s|\n", dashes);
+   platform_default_log("| Allocator Stats                                                   |\n");
+   platform_default_log("|%s|\n", dashes);
+   // clang-format on
+
+   char   bytes_to_str[SIZE_TO_STR_LEN];
+   size_t nbytes = (al->stats.curr_allocated * al->cfg->io_cfg->extent_size);
+   size_to_fmtstr(bytes_to_str, sizeof(bytes_to_str), "(%s)", nbytes);
+
    platform_default_log(
-      "| Currently Allocated: %12lu extents (%4luGiB)          |\n",
+      "| Currently Allocated: %12lu extents %-14s          |\n",
```

Comment:
This change prints a more accurate value, rather than just printing as n-`GiBs`.

Reply:
@rtjohnso - I can't tell from your comment if this item is (in your words) "a strongly preferred changes" or "a required change".

The 2-step process does not bother me. I am not very keen to use multiple instances of `size_str()` in one function due to the potential stack over-use issue we discussed previously.

What remains is formatting and indentation. The 'problem', I think, is also that we have carefully constructed these tabular outputs (which are nice to read!), so I have to work a bit extra hard to get the alignment right.

Also, I don't think `"(       ~1.6 GiB)"` reads very well. It spoils the aesthetics of the carefully constructed aligned tabular outputs.

One small thing I could do for just the two lines reporting currently allocated and max allocated extents is to drop the enclosing `( )`, but that would make the output of this size in unit-specifiers field inconsistent with the rest of the output generated in this function. So, I'm not so keen on that either.

Let's talk off-line ... to see how best to resolve this.